<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>480</width>
      <height>516</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="0">
        <widget class="QRadioButton" name="change_cof_a">
         <property name="text">
          <string>Изменить значение коэффициента а</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QRadioButton" name="change_roots">
         <property name="text">
          <string>Изменить значение корней </string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>n</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>коэфф. a</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>корни</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="line_counter"/>
           </item>
           <item>
            <widget class="QLineEdit" name="line_cof_a"/>
           </item>
           <item>
            <widget class="QLineEdit" name="line_root"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="9" column="1">
        <widget class="QRadioButton" name="polynom_1">
         <property name="text">
          <string>1-го вида</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QRadioButton" name="polynom_2">
         <property name="text">
          <string>2-го вида</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Вывести полином</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>индекс корня</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>новое значение</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="line_index_root"/>
           </item>
           <item>
            <widget class="QLineEdit" name="line_value"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="resize_cof">
         <property name="text">
          <string>Изменить размер массива корней</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QRadioButton" name="polynom_x">
         <property name="text">
          <string>Найти значение полинома в точке х</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="real_button">
         <property name="text">
          <string>вещественные числа</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Выберите:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="line_size"/>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="polynom_n">
         <property name="text">
          <string>Создать полином n-й степени</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="line_cof_a_2"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="line_x"/>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="complex_button">
         <property name="text">
          <string>комплексные числа</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Подтвердить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
